### PR TITLE
[docs] Document anyOf expression

### LINKED ISSUE #3503: Document `anyOf` expression

### PR DESCRIPTION
**Pull request by anastasi-n**
Closes #3503

### CODE REVIEW

File: docs/modules/commons/pages/expressions.adoc [hunk 1]
```diff
@@ -740,6 +740,46 @@ Gets the substring after the first occurrence of a separator. The separator is n
...
+----
+#{anyOf(*$list*)}
+----
+
+* *`$list`* - comma-separated list of values
+
+.Examples of the expressions picking a random value
+|===
+|Expression |Result
+
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:757]: add some simple example:
```
|`#{anyOf(1, 2, 3)}`
|either `1`, or `2`, or `3`

```

**Review by valfirst — APPROVED** [review]

File: docs/modules/commons/pages/expressions.adoc [hunk 2]
```diff
@@ -740,6 +740,46 @@ Gets the substring after the first occurrence of a separator. The separator is n
...
+|`123`
+
+|`#{anyOf(1\,2)}`
+|`1,2`
+
+|`#{anyOf()}`
+|
+
+|`#{anyOf( )}`
+|
```
Comment (uarlouski) [inline docs/modules/commons/pages/expressions.adoc:771]: is it space or empty string?

Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:771]:
```suggestion
|<empty string>
```

File: docs/modules/commons/pages/expressions.adoc [hunk 3]
```diff
@@ -740,6 +740,46 @@ Gets the substring after the first occurrence of a separator. The separator is n
...
+|`1,2`
+
+|`#{anyOf()}`
+|
+
+|`#{anyOf( )}`
+|<empty string>
+
+|`#{anyOf(,)}`
+|
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:774]:
```suggestion
|<empty string>
```

File: docs/modules/commons/pages/expressions.adoc [hunk 4]
```diff
@@ -740,6 +740,46 @@ Gets the substring after the first occurrence of a separator. The separator is n
...
+
+|`#{anyOf( )}`
+|<empty string>
+
+|`#{anyOf(,)}`
+|<empty string>
+
+|`#{anyOf(, )}`
+|
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:777]:
```suggestion
|<empty string>
```

File: docs/modules/commons/pages/expressions.adoc [hunk 5]
```diff
@@ -740,6 +740,46 @@ Gets the substring after the first occurrence of a separator. The separator is n
...
+|either `1`, or `2`, or `3`
+
+|`#{anyOf(123)}`
+|`123`
+
+|`#{anyOf(1\,2)}`
+|`1,2`
+
+|`#{anyOf()}`
+|
```
Comment (valfirst) [inline docs/modules/commons/pages/expressions.adoc:768]:
```suggestion
|<empty string>
```

**Pull request merged**